Description: musl uses 64-bits time_t on 32-bits arches
Upstream: no
diff --git a/src/Settings.cpp b/src/Settings.cpp
index fd1d314..b1853a6 100644
--- a/src/Settings.cpp
+++ b/src/Settings.cpp
@@ -1054,7 +1054,7 @@ void Settings_ModalOptions (struct thread_Settings *mExtSettings) {
 	    bail = true;
 	} else if (isTxStartTime(mExtSettings) || isTxHoldback(mExtSettings)) {
 	    Timestamp now;
-	    long nowsecs = now.getSecs();
+	    time_t nowsecs = now.getSecs();
 	    // fill out the formats in the event they are needed per an time error
 	    char start_timebuf[80];
 	    struct tm ts = *localtime(&mExtSettings->txstart_epoch.tv_sec);
